packages/mobile/src/screens/DetailsScreen.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import {
+  SafeAreaView,
+  StyleSheet,
+  ScrollView,
+  View,
+  Text,
+  Button,
+  StatusBar,
+} from 'react-native';
+
+const DetailsScreen = ({navigation}) => {
+    return (
+      <View style={{flex: 1, alignItems: 'center', justifyContent: 'center'}}>
+        <Text>Details Screen</Text>
+        <Button
+          title="Back to Home"
+          onPress={() => navigation.navigate('Home')}
```

Comment:
funkcja strałkowa w render jest anty-wzorcem. Dlatego że spowalnia render bo tworzy nowe funkcje przy każdym renderze co jest marnotrastwem zasobów, co gorsze komponenty którym przekazujemy propsy z każdym przekazaniem nowego liczą się ponownie a gdy tworzymy nową funkcje z renderem to chodz ona sie nie zmienia to komponent mysli ze to nowy props i liczy sie ponownie (!Uwaga zapamietać pytają o anty-wzorce czesto na rozmowach)